src/components/Faq/Faq.vue

```diff
@@ -0,0 +1,339 @@
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "Can I register a project that is raising funds?",
+          ans: "Projects that are being fund-raised can be registered.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "Why we choose cryptocurrency?",
+          ans: "Only through cryptocurrencies is a complex, unique and sustainable system like Dev Protocol currently possible. The token emission and all the incentives we create are only possible using our own token.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "How to use stakes.social?",
+          ans: "It is really straight forward, visit stakes.social, connect your wallet find the project you like the most and stake DEV tokens to sustainably fund them.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "How to promote my OSS project to get sponsor on stakes.social?",
+          ans: "The best way is by sharing DEV within your own community. Explain to them how Dev Protocol allows sustainable fundingl, how patrons also receive rewards, and how it's different from a simple donation. The bigger the project gets, the more you have the chance for others outside your community to find your project.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "How is the gas price calculated?",
+          ans: "The gas fees depends on the usage of the Ethereum network. Every operation that happens on this network requires gas to be executed, so the bigger the demand for the network, the bigger the gas fees fill be.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "How can creators make money with Dev Protocol?",
+          ans: "Creators have to onboard their projects on stakes.social, Patrons can Stake DEV tokens on their project's pool and sustainably fund them via DEV Token Rewards.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "Can Property Token's name be changed ?",
+          ans: "To rename or rename a PropertyToken, you can look for the address of the Property on Etherscan and execute changeName / changeSimbol from the Write Contract of the Contract.",
+          lang: "English",
+          tag: "creators",
+        },
+        {
+          ques: "Is the DEV token a security?",
+          ans: "We don't believe DEV tokens are considered securities for multiple reasons. First even if creators don't hold any DEV, they can receive DEV as their reward in compensation for their activities. While there is a staking reward built in to incentivize interaction. Users that stake on creators are doing so to signal their importance rather than an expectation of a reward. The DEV token in our eyes is strictly a utility token used to aid the ecosystem.",
```

Comment:
```suggestion
          ans: "We don't believe DEV tokens are considered securities for multiple reasons. First, even if creators don't hold any DEV, they can receive DEV as their reward in compensation for their activities. While there is a staking reward built in to incentivize interaction. Users that stake on creators are doing so to signal their importance rather than an expectation of a reward. The DEV token in our eyes is strictly a utility token used to aid the ecosystem.",
```